### PR TITLE
yamllint formatting cleanup for ocp-workload-dm7-realtime-event-decisioning

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Create a yamllint file to enforce formatting rules
- Make changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp-workload-event-decisioning

### CODE REVIEW

File: ansible/roles/ocp-workload-dm7-realtime-event-decisioning/tasks/wait_for_deploy.yml [hunk 1]
```diff
@@ -6,25 +6,34 @@
...
-- name: "Wait for following deployments to become ready: {{pod_to_wait}}"
+- name: "Wait for following deployments to become ready: {{ pod_to_wait }}"
   command: 'oc get rc -o json -n "{{ ocp_project }}"'
   register: rc_state
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rc_state.stdout |from_json |json_query("items[? ((status.readyReplicas == \"\" || status.readyReplicas == ''`0`'') && (metadata.annotations.\"openshift.io/deployment.cancelled\" != ''true''))].metadata.annotations.\"openshift.io/deployment-config.name\"") |intersect(pod_to_wait) |length == 0'
+  until: >-
+    rc_state.stdout | from_json | json_query("items[? ((status.readyReplicas == "" || status.readyReplicas
```
Comment (stencell) [inline ansible/roles/ocp-workload-dm7-realtime-event-decisioning/tasks/wait_for_deploy.yml:33]: this has character escapes removed, so must be tested.

**Review by stencell — CHANGES_REQUESTED** [review]
Some small code changes, must be tested.

File: ansible/roles/ocp-workload-dm7-realtime-event-decisioning/tasks/wait_for_deploy.yml [hunk 2]
```diff
@@ -6,25 +6,34 @@
...
   register: rc_state
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rc_state.stdout |from_json | json_query("items[? (kind == ''ReplicationController'')].metadata.annotations.\"openshift.io/deployment-config.name\"") |intersect(item) |length >= 1'
-  loop: "{{pod_to_wait}}"
+  until: >-
+    rc_state.stdout | from_json | json_query("items[?
+    (kind == ''ReplicationController'')].metadata.annotations."openshift.io/deployment-config.name"")
+    | intersect(item) |length >= 1
```
Comment (jkupferer) [inline ansible/roles/ocp-workload-dm7-realtime-event-decisioning/tasks/wait_for_deploy.yml:23]: This logic is hard to follow, it would be better if the json query were a separate var referenced in the filter.

File: ansible/roles/ocp-workload-dm7-realtime-event-decisioning/tasks/wait_for_deploy.yml [hunk 3]
```diff
@@ -6,25 +6,34 @@
...
   command: 'oc get rc -o json -n "{{ ocp_project }}"'
   register: rc_state
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rc_state.stdout |from_json |json_query("items[? ((status.readyReplicas == \"\" || status.readyReplicas == ''`0`'') && (metadata.annotations.\"openshift.io/deployment.cancelled\" != ''true''))].metadata.annotations.\"openshift.io/deployment-config.name\"") |intersect(pod_to_wait) |length == 0'
+  until: >-
+    rc_state.stdout | from_json | json_query("items[? ((status.readyReplicas == "" || status.readyReplicas
+    == ''`0`'') && (metadata.annotations."openshift.io/deployment.cancelled"
+    != ''true''))].metadata.annotations."openshift.io/deployment-config.name"")
```
Comment (jkupferer) [inline ansible/roles/ocp-workload-dm7-realtime-event-decisioning/tasks/wait_for_deploy.yml:35]: Double quote here is almost certainly an error.

**Review by jkupferer — CHANGES_REQUESTED** [review]
